docs/tutorials/quick_install.md

```diff
@@ -187,15 +148,15 @@ During this task, you will prepare the **config.yml** file to configure the envi
 
 1. Run the Terraform plan script to validate changes to be executed.
     ```bash
-    ./build.sh -a plan
+   ./build.sh -a plan
     ```
     If no errors have been detected you can proceed to execute the following code to start the deployment.
     ```bash
-    ./build.sh -a apply 
+   ./build.sh -a apply 
```

Comment:
shouldn't use bicep here ? if so please add "-l bicep"

Reply:
Yes, I added the parameter now.